src/environment.js

```diff
@@ -63,6 +63,8 @@ class Environment {
     switch (this.ci) {
       case 'gitlab':
         return `gitlab/${this._env.CI_SERVER_VERSION}`;
+      case 'github-action':
+        return `github-action/${this._env.PERCY_GITHUB_ACTION || 'unkown'}`;
```

Comment:
I think we could remove `github-action/`? since it's now a CI version?

Reply:
We could probably simplify it as `github` and leave it in the UA as `github/exec-action/0.1.0` or `github/unkown` in the case of somebody not using an official action. 👍